Sources/RedCat/Observers.swift

```diff
@@ -8,58 +8,34 @@
 import Foundation
 
 
```

Comment:
Wait a sec, is this file still in use?

Reply:
ObservableStore.swift line 36, if I get you right

Reply:
Ahhhh, ok. Essentially this is because you distinguish between store delegates and store observers. Given my concerns about sharing states, there's an opportunity to merge the two features, right? If delegates just watch for ```storeDidChange``` like they originally did, they could just subscribe to the store's publisher.